idemp-rest-test/testing/src/utils/Utils.ts

```diff
@@ -86,6 +86,32 @@ export async function createProduct(valueObject: ValueObject) {
 	}
 }
 
+/**
+ * Create a product (don't really have an ideal method for this at the moment - have to go through items).
+ * If a product already exists on the value object, this won't do anything.
+ * @param valueObject The value object containing information to create the entity
+ * @returns Nothing
+ */
+export async function createProductInventory(valueObject: ValueObject) {
```

Comment:
This looks to do the same thing as the `createProduct` method, so can you remove this?